src/ConnectionLocator.php

```diff
@@ -10,12 +10,29 @@
 
 namespace Atlas\Pdo;
 
+/**
+ * @phpstan-type connectionStore array{
+ *      DEFAULT: ?Connection,
+ *      READ: array<string, ?Connection>,
+ *      WRITE: array<string, ?Connection>
+ * }
+ * @phpstan-import-type logEntryType from Connection
+ */
 class ConnectionLocator
 {
+    /**
+     * @var string
+     */
     public const DEFAULT = 'DEFAULT';
```

Comment:
I don't think it's a BC break to add `string` typehints on these ... though it might not be 8.0 compatible. Thoughts?